TheraLang.DAL/Repository/Repository.cs

```diff
@@ -9,46 +9,77 @@ namespace TheraLang.DAL.Repository
 {
     public class Repository<TEntity> : IRepository<TEntity> where TEntity : class
     {
+        private DbSet<TEntity> DbSet { get; }
```

Comment:
Its used only in repository
```suggestion
        private readonly DbSet<TEntity> _dbSet;
```